Jenkinsfile_CNP

```diff
@@ -7,6 +7,6 @@ def product = 'div'
 withInfraPipeline(product) {
 }
 
-onMaster {
+withInfraPipeline {
```

Comment:
This won't work, as it's duplicating the `withInfraPipeline`

Reply:
You just need one block
```

withInfraPipeline(product) {
    enableSlackNotifications("#div-dev")
}
```

Reply:
Yeah oops my bad